src/Mod/Path/PathScripts/PathFixture.py

```diff
@@ -129,7 +129,7 @@ def Activated(self):
 import Path
 import PathScripts
 from PathScripts import PathUtils
-prjexists = False
+prjexists_variable_is_unused = False
```

Comment:
Afaict this was never used from the point when it landed. I tried finding the commit in the original branch, but there were ~250 iirc and some had incorrect commits and I gave up.